src/systemtest/resources/JenkinsShellScripts/TCTScript.txt

```diff
@@ -1 +1,61 @@
-echo "progress ongoing"
\ No newline at end of file
+import groovy.json.JsonSlurper
+def baseUrl = "<REMREM_BASE_URL>";
+
+////////////////////////////////////////////////////////////////////////////////////////
+///////////////////////////////////////CLM1/////////////////////////////////////////////
+////////////////////////////////////////////////////////////////////////////////////////
+
+def uriPathTCT1 = baseUrl + '/generateAndPublish?mp=eiffelsemantics&amp;msgType=EiffelTestCaseTriggeredEvent';
```

Comment:
The naming of the JenkinsShellScripts are a bit confusing... For example it's easy to mistake the TCTScript with TCSTCFScript :) Although I understand the benefit of shorter names, perhaps writing it out a bit more would be helpful when there are so many abbreviations.

Reply:
The thing is, some of these script contains several events so writing the ArtC2 out would be even more confusing as it sends more events than just artifact created. I named it after the ArtifactRules.png in the documentation so it should be looked on as step names and not event names.